geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -14,13 +14,24 @@ use {
 pub struct LoadedGeyserPlugin {
     name: String,
     plugin: Box<dyn GeyserPlugin>,
+    // NOTE: While we do not access the library, the plugin we have loaded most
+    // certainly does. To ensure we don't SIGSEGV we must declare the library
+    // after the plugin so the plugin is dropped first.
```

Comment:
Would be nice if rust let us (easily) specify drop order, without all the `ManuallyDrop` noise.

Happy to just rely on the implicit drop order (drop in decl order)

Reply:
I _think_ it might be stabilized:

https://github.com/rust-lang/rfcs/blob/246ff86b320a72f98ed2df92805e8e3d48b402d6/text/1857-stabilize-drop-order.md

Reply:
Yeah, the drop order is stabilized. Just not a huge fan of having safety rely on implicit things such as that.